examples/language_model/bert/predict_glue.py

```diff
@@ -89,6 +93,47 @@ def create_predictor(cls, args):
             # set XPU configs accordingly
             config.enable_xpu(100)
         config.switch_use_feed_fetch_ops(False)
+        if args.use_tensorrt:
+            config.enable_tensorrt_engine(
+                workspace_size=1 << 30,
+                precision_mode=inference.PrecisionType.Float32,
+                max_batch_size=1,
+                min_subgraph_size=5,
+                use_static=False,
+                use_calib_mode=False)
+            print("Enable TensorRT is: {}".format(
+                config.tensorrt_engine_enabled()))
+
+            names = [
+                "input_ids",
+                "token_type_ids",
+                "tmp_4",
+                "unsqueeze2_0.tmp_0",
+            ]
+            min_input_shape = [1, 1]
```

Comment:
Add Comment here to demonstrate the reason why we need set min and max input shape.